src/views/MainView.vue

```diff
@@ -9,6 +9,7 @@
 					<CallView :token="token" />
 				</template>
 			</TransitionWrapper>
+			<PollViewer />
```

Comment:
Maybe just keep it in Poll component ?

Reply:
What do you mean?
They've asked for a while to be separated, and we need the modal to be mounted on ChatView/CallView level to be able to use them in both